src/platform/shared.c

```diff
@@ -58,6 +59,7 @@ SDL_Color gPalette[256];
 uint32 gPaletteHWMapped[256];
 
 bool gSteamOverlayActive = false;
+Uint32 gOpenRCT2EventType;
```

Comment:
I was thinking that this could refer to all custom events we may add to OpenRCT2. I had some plans for implementing a custom event to allow for re-adding the tutorial by having an event for clicking on a UI element by ID (instead of a coordinate like the existing click events).

I'm actually going to rewrite the event thing a little bit. We should probably actually use multiple event types instead of using the code to specify the type (this can allow for the code attribute to be used within an event, such as for specifying a UI element as I mentioned).